infrastructure/network/netadapter/router/router.go

```diff
@@ -45,7 +46,7 @@ func (r *Router) AddIncomingRoute(messageTypes []appmessage.MessageCommand) (*Ro
 // AddIncomingRouteWithCapacity registers the messages of types `messageTypes` to
 // be routed to the given `route` with a capacity of `capacity`
 func (r *Router) AddIncomingRouteWithCapacity(capacity int, messageTypes []appmessage.MessageCommand) (*Route, error) {
-	route := newRouteWithCapacity(capacity)
+	route := newRouteWithCapacity("", capacity)
```

Comment:
AddIncomingRouteWithCapacity should also accept a name and pass it to newRouteWithCapacity.